pkg/printers/internalversion/printers_test.go

```diff
@@ -6206,6 +6208,322 @@ func TestPrintFlowSchema(t *testing.T) {
 	}
 }
 
+func TestPrintDeviceClass(t *testing.T) {
+
+	tests := []struct {
+		deviceClass resourceapis.DeviceClass
+		expected    []metav1.TableRow
+	}{
+		{
+			deviceClass: resourceapis.DeviceClass{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:              "deviceclass",
+					CreationTimestamp: metav1.Time{Time: time.Now().Add(1.9e9)},
+				},
+				Spec: resourceapis.DeviceClassSpec{
+					SuitableNodes: &api.NodeSelector{
+						NodeSelectorTerms: []api.NodeSelectorTerm{{
+							MatchExpressions: []api.NodeSelectorRequirement{{
+								Key:      "foo",
+								Operator: api.NodeSelectorOpExists,
+							}},
+						}},
+					},
+				},
+			},
+			// Columns: Name, Age
+			expected: []metav1.TableRow{{Cells: []interface{}{"deviceclass", "0s"}}},
```

Comment:
Could you please add a test case for Empty creation timestamp that is translated to `<unknown>`?

Reply:
I have added in all cases